tests/network/sriov.go

```diff
@@ -687,14 +687,13 @@ func findIfaceByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineIns
 			return false, err
 		}
 
-		ifaceName, err = getInterfaceNameByMAC(vmi, mac)
-		if err != nil {
+		if ifaceName = getInterfaceNameByMAC(vmi, mac); ifaceName == "" {
```

Comment:
Please drop this change, it is not related to what this PR  is aimed for.
It is also revealing a stronger problem, where the status of the VMI is not reflecting the guest agent info post migration. (see #7893)

Reply:
I changed `getInterfaceNameByMAC()` to not return an error because it never propagated and was ignored 
in order to keep` wait.Poll()` going. It looked odd to leave it like this.

Reply:
I see that on #7893 is changed to draft and is the timeout and the conditionally keep the polling based on `getInterfaceNameByMAC()` result.

It doesn't seem that these PR changes contradict with #7893, WDYT?

Reply:
I removed those changes, I suggest you add these at #7893.